pkg/app/app.go

```diff
@@ -25,10 +26,16 @@ var (
 	// routed.
 	NotFound UI = &notFound{}
 
-	routes = make(map[string]UI)
-	uiChan = make(chan func(), 256)
+	routes   = make(map[string]UI)
+	routesRe = make([]regexRoute, 0)
+	uiChan   = make(chan func(), 256)
 )
 
+type regexRoute struct {
+	re *regexp.Regexp
```

Comment:
Same here, can we rename this to `regexp`.

Reply:
good suggestion. will do

Reply:
agreed